README.md

```diff
@@ -47,6 +47,14 @@ The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to po
 
 Using FTP transfer the contents of the osisoft_connector directory to the /usr/ directory of your eWON via FTP.  If successful you should have a css directory, js directory, config.html, ConnectorConfig.json, flexy-osisoft-connector.jar, and jvmrun located in the /usr/ directory of the Flexy.  You must also follow all steps in the Certificates section below.
 
+### Tag Configuration
+
+The connector reads from the tags historical logs to write to OSIsoft. For best results enable the Logging Deadband feature. Again, the historical logs are sent through the connector to OSIsoft. The settings for the historical log deadband is the same as the settings for the OSIsoft connector deadband.
+
+#### Deadband
+
+With a deadband of 1, the tag value will need to change by 1 before being logged. If you have logging deadband set to -1, we recommend you set the time interval to something high. Set the deadband to whatever your lowest unit of interest is.
```

Comment:
Where it says "If you have logging deadband set to -1 ...", I would make it include that -1 also means disabled. "If you have logging deadband disabled (set to -1) ..."